neps/nep-0452.md

```diff
@@ -0,0 +1,220 @@
+///   the newly created account.
+///
+/// Returns `true` if the claim was successful meaning the `new_account_id` was created and all assets were sent to it.
+function create_account_and_claim(new_account_id: string, new_public_key: string) -> Promise<boolean>
+```
+
+## Example Scenarios
+
+*Pre-requisite Steps*: Linkdrop creation:
+The linkdrop creator that has an account and some $NEAR:
+
+- creates a keypair locally (`pubKey1`, `privKey1`). (The keypair is not written to chain at this time)
+- calls a method, passing the `pubKey1` and desired $NEAR amount as arguments, to a contract that implements the linkdrop standard.
+- The contract will map the `pubKey1` to the desired balance for the linkdrop.
+- The contract will then add the `pubKey1` as a function call access key with the ability to call `claim` and `create_account_and_claim`. This means that anyone with the `privKey1` (see above), can sign a transaction on behalf of this contract (signer id set to contract id) with a function call to call one of the mentioned functions to claim the assets.
+
+### Claiming a linkdrop without a NEAR Account
+
+A user with *no* wallet that is claiming the assets associated with the public key:
+
+- generates a new keypair (`pubKey2`, `privKey2`) locally. (The keypair is not written to chain at this time)
+- chooses a new account ID such as benji.near.
+- signs a transaction on behalf of the linkdrop contract (contract is set to signer_id) using `privKey1`. The method will be `create_account_and_claim`.
+- the args of this function call will contain both `pubKey2` (which will be used to create a full access key for the new account) and the account ID itself.
+- the linkdrop contract will create the new account and transfer the funds to it alongside any other assets.
+- the linkdrop contract will delete the access key associated with `pubKey1` so that it cannot be used again.
+- the user will be able to sign transactions on behalf of the new account using `privKey2`.
+
+### Claiming a linkdrop with a NEAR Account
+
+A user with an *existing* wallet that is claiming the assets associated with the public key:
+
+- signs a transaction on behalf of the linkdrop contract (contract is set to signer_id) using `privKey1`. The method will be `claim`.
+- the args of this function call will simply contain the user's existing account ID.
+- the linkdrop contract will transfer the funds to that account alongside any other assets.
+- the linkdrop contract will delete the access key associated with `pubKey1` so that it cannot be used again.
+
+
+## Reference Implementation
+
+Below are some references for linkdrop contracts. A specific implementation of this standard can be seen in [this pull request](https://github.com/near/near-linkdrop/pull/24).◊
```

Comment:
Well, it was an extension PR, so I would drop this reference as it might be misleading.